languages/go/config.json

```diff
@@ -141,7 +140,7 @@
         "slug": "nil",
         "uuid": "7b265cc8-8f88-4de1-b653-10a384e6c7e2",
         "concepts": ["nil"],
-        "prerequisites": ["basics", "maps", "pointers", "channels"]
+        "prerequisites": ["basics"]
```

Comment:
should we include "types" here as well?

Reply:
You are fast 😄 

"types" was replaced by `basics`. At least in the current plan (which only exists in my mind...)